src/scripts/setup.sh

```diff
@@ -12,6 +12,25 @@ Install() {
     if [ "$UID" = "0" ]; then export SUDO=''; else export SUDO='sudo'; fi
     $SUDO find /etc/datadog-agent/conf.d/ -iname "*.yaml.default" -delete
     $SUDO service datadog-agent start
+
+    set +e
+    attempts=0
+    $SUDO datadog-agent health
+    exit_code=$?
+    until [[ $attempts -ge 10 ||  $exit_code -eq 0 ]]; do
+        attempts=$((attempts+1))
+        sleep_time=$(( attempts*10 < 60 ? attempts*10 : 60 ))
```

Comment:
This seems like a very long time? How long does it take to get the agent up & running usually? I'd try to avoid having to wait up to a minute to advance to the next step if the agent takes 1-2 minutes usually.

Reply:
It seems to vary a lot in most of my test it starts working around attempt 4 or 3 so that is around 1 to 1min 30s. I can cap the maximum sleep time at 20 seconds if you think that be better.